src/reactions/MessageReactionList.js

```diff
@@ -83,38 +85,58 @@ type Props = $ReadOnly<{|
  * screen first appears.
  */
 class MessageReactionList extends PureComponent<Props> {
+  componentDidMount() {
+    if (this.props.message === undefined) {
+      const { messageId } = this.props.navigation.state.params;
+      logging.warn(
+        `MessageReactionList unexpectedly created without props.message; message ${messageId} is missing in state.messages`,
```

Comment:
Good to know. Especially since there's a known cause of this error that won't be fixed until https://github.com/zulip/zulip-mobile/pull/3868/commits/25000aa5f9d1a465ca2a888b1a6df06f0c0f0ee6 lands.

Reply:
(I've made the requested change.)